doc/host-setup.md

```diff
@@ -0,0 +1,64 @@
+# Preparing the host for DRBD
+
+Piraeus depends on DRBD for data replication across multiple nodes. [DRBD] is a kernel module that must be loaded on
+every node that should access the storage. Depending on your exact host OS and set up, you can choose from one of the
+following options to load DRBD:
+
+[DRBD]: https://github.com/LINBIT/drbd/
+
+## Build and load DRBD using the Kernel Module Injection Image (Preferred)
+
+Every satellite container created by the operator will use an [InitContainer] to ensure DRBD is available. This init
+step can also compile and load DRBD directly. To build DRBD, the container needs access to the kernel sources. It will
+look for the sources on the host under `/usr/src` and `/lib/modules/$(uname -r)`. Most distributions provide a convenient package
+for the kernel source:
+
+* Ubuntu: `apt-get install linux-headers-$(uname -r)`
+* CentOS: `yum install kernel-devel-$(uname -r)`
```

Comment:
For me it was called `kernel-devel` on CentOS. I also installed `kernel-headers`, I don't know if it's needed but it seemed obvious to have besides the source files. 

I also installed `cryptsetup` to get the greenLUKS checkmarks in Linstor output but it didn't work, it's always unavailable.

Reply:
`kernel-devel` always tracks the latest kernel version, which may not be the version currently running, so I added the `$(uname -r)` shortcut.

Haven't checked on LUKS support in a while, I guess that's a separate issue.